.travis.yml

```diff
@@ -3,23 +3,30 @@ language: php
 git:
     depth: 1
 
-sudo: false
-
 matrix:
     fast_finish: true
     include:
         - php: 5.6
+          dist: xenial
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
         - php: 7.0
+          dist: xenial
           env: COMPOSER_FLAGS="--prefer-stable"
         - php: 7.1
+          dist: bionic
           env: COMPOSER_FLAGS="--prefer-stable"
         - php: 7.2
+          dist: bionic
           env: COMPOSER_FLAGS="--prefer-stable"
         - php: 7.3
+          dist: bionic
+          env: COMPOSER_FLAGS="--prefer-stable"
+        - php: 7.4
+          dist: bionic
```

Comment:
why explicit `dist` ?

Reply:
So that things don't break when travis next change the default dist. 16.04 or 18.04 would both be fine here, but not for the older php versions.